### DISCUSSION TITLE
Algolia Search integration

### DISCUSSION THREAD

**Discussion by xnhinzkyx** [breezefront/community#201] [3. Integrations]
Hello,

I really appreciate it if you can integrate Algolia Search to breeze. Algolia Search is one of the most popular and powerful search tool not only for Magento 2.

Here is the link of the extension:
https://www.algolia.com/search-solutions/adobe-commerce-magento/
https://github.com/algolia/algoliasearch-magento-2

Thanks!

**Comment by vovayatsyuk** [comment]
Just tagged initial release: https://github.com/breezefront/module-breeze-algoliasearch

**Reply by niravGV** [reply]
Hello @vovayatsyuk 

What are the feature work with this initial release of the module?

Thank you

**Reply by vovayatsyuk** [reply]
Hello,

Do you have a question about some functionality? Have you tried the module?

I don't remember what features have algolia search besides actual search. Please clarify what features you refer to.

**Reply by niravGV** [reply]
Hello @vovayatsyuk

Yes like the Instance search (quick search) & Category and Search page with the Algolia Widget to display the list of the Products and filters works as per the Js of Algolia.

Does this two feature working with this module, i will surely try it tomorrow and update here.

Thank you

**Reply by vovayatsyuk** [reply]
> Category and Search page with the Algolia Widget

Didn't know about this feature, so I guess only the quick search is integrated.

**Comment by divyeshdusara867** [comment]
Hello @vovayatsyuk
I need to compatible recommendation product of algolia like Frequently Bought Together, Related Products and Trending Items with breeze front on product page. so is this module is compatible ? I did tried with its but not working on product page

**Reply by sweety-dd** [reply]
Hello @divyeshdusara867 Have you find any solution for product page Related Products, Frequently Bought Together product section?
